python/py-m2crypto/Portfile

```diff
@@ -17,9 +17,9 @@ homepage           https://pypi.python.org/pypi/${python.rootname}
 
 master_sites       pypi:m/${python.rootname}/
 distname           M2Crypto-${version}
-checksums          md5     cced2e2273dbb4a6094c96ae20d1b9e4 \
-                   rmd160  1cbdfd7312b28468d7568f227bd35dc4ec0e61a1 \
-                   sha256  a0fea2c5ab913e42864d1ccbaee5878c23886368b606e923609fda4ce37d26c0
+checksums          md5     45fb8bf2b64cfe9d9d72e332de339a2c \
+                   rmd160  2ccbcbe161d9e58015a5d7d720333d75ed9239c1 \
```

Comment:
FYI, we no longer include md5 hashes in our Portfiles, but we do include "size".

Reply:
@pmetzger, @larryv asked me to include md5 as an additional check;  see his review comment on #44. I've since added it to both of the PyPI ports I maintain.

As for the "size", can you point me to a reference I can use as a guide? `port distfiles py36-m2crypto` does not show that property and I can't find any mention in the Portfile Reference page. I'm not sure how to add it.

Reply:
I can't find that comment in #44, but from now on, you shouldn't include MD5s, but you should include "size". Just grep around in the ports tree and you'll see examples. You don't need to remove them as of yet, but when you update ports, leave them out, and just include the new cryptographic hashes (rmd160, sha256).

One way to generate these automatically is using `port -v checksum`, note that it will not spit out a  parameter unless there is one already, so I often include `size 0` for example when creating a Portfile so that `port -v checksum` will generate one for me.